docs/docs/cmd/graph/subscription/subscription-add.mdx

```diff
@@ -128,6 +131,7 @@ m365 graph subscription add --resource user --changeTypes "created,updated,delet
     "expirationDateTime": "2023-05-23T18:23:45.935Z",
     "creatorId": "0649d0bd-53dc-4e1d-a357-76f1d92d447b",
     "includeResourceData": null,
```

Comment:
I do not think we should update this. The `includeResourceData` is part of the request/response from the API.